test/src/test/resources/fixtures/query with two services sharing a namespaced field. fragment query.yml

```diff
@@ -0,0 +1,136 @@
+name: query with two services sharing a namespaced field. fragment query
+enabled:
+  nextgen: false
+  current: true
+overallSchema:
+  Issues: |
+    directive @namespaced on FIELD_DEFINITION
+
+    type Query {
```

Comment:
What happens when you have 2 fields with same name/path in 2 different services both with the `@namespaced` directive?
This is probably illegal in Nadel, but if it is legal, maybe we could have a test covering that.

Reply:
@felipe-gdr can you give me a snippet with the code you are describing to make sure we are on the same page?

```graphql
service A {
  type Query {
    a: AQuery @namespaced
  }
  ...
}

service B {
  type Query {
    a: BQuery @namespaced
  }
  ...
}
```

is that what you mean?

Reply:
I was thinking on something like:

```
   service A {
     type Query {
        issue: IssueQuery @namespaced
      }
      type IssueQuery {
        getIssue: Issue
      }
   }   

   service B {
     type Query {
        issue: IssueQuery @namespaced
      }
      type IssueQuery {
        search: SearchResult
      }
   }   
```

Which is not support by Nadel, I believe -> the `IssueQuery` type is being declared twice and would need to be merged

Reply:
well actually you can't even do that

```graphql
   service A {
     type Query {
        issue: String
      }
   }   

   service B {
     type Query {
        issue: String
      }
   }   
```
Because you can't have duplicate fields on a type. This schema will blow up here `graphql.nadel.schema.OverallSchemaGenerator.buildOverallSchema` with an error `SchemaProblem{errors=[The type 'Query' [@-1:-1] has declared a field with a non unique name 'issue']}` 
I don't think we should be testing it here as it's a fundamental `graphql-java` capability and doesn't have anything to do with `@namespaced` feature we are working on here